graphql/src/use_cases/sample.py

```diff
@@ -5,7 +5,7 @@
 from src.entities.common import Edge
 from src.entities.echo import BoringWordException
 from src.use_cases.adapter_interfaces.word_generation import WordGenerationInterface
-from src.use_cases.edge_cursor import EdgeCursor
+from src.use_cases.helpers.edge_cursor import EdgeCursor
```

Comment:
Not sure why you are nesting under helpers here, I would consider the way we create our cursor a use case and not a helper.

Reply:
It is already under use_cases. helpers is a folder within use_cases, to help differentiate between types of use cases.